src/OrbitGl/App.cpp

```diff
@@ -1445,8 +1445,23 @@ void OrbitApp::SetVisibleFunctionIds(absl::flat_hash_set<uint64_t> visible_funct
   NeedsRedraw();
 }
 
-[[nodiscard]] bool OrbitApp::IsFunctionVisible(uint64_t function_address) {
-  return data_manager_->IsFunctionVisible(function_address);
+bool OrbitApp::IsFunctionVisible(uint64_t function_address) {
+  if (data_manager_->IsFunctionVisible(function_address)) {
+    return true;
+  }
+
+  // Filtering for manually instrumented scopes is not yet supported. All "Orbit" functions
```

Comment:
Nit: Can you add a bug to track this, and convert this into an `TODO (b/...): ....`?

Reply:
Done.